blueoil/pre_processor.py

```diff
@@ -19,7 +19,24 @@
 from blueoil.data_processor import Processor
 
 
-def resize(image, size=[256, 256]):
+NEAREST = "NEAREST"
+BOX = "BOX"
+BILINEAR = "BILINEAR"
+HAMMING = "HAMMING"
+BICUBIC = "BICUBIC"
+LANCZOS = "LANCZOS"
+
+RESAMPLE_METHODS = {
+    NEAREST: PIL.Image.NEAREST,
```

Comment:
This line is better than `"NEAREST": PIL.Image.NEAREST` ?
When we add resize algorithm, we need to change 2 lines.